NuKeeper.BitBucket/BitbucketRestClient.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using NuKeeper.Abstractions.Logging;
+using NuKeeper.BitBucket.Models;
+
+namespace NuKeeper.BitBucket
+{
+    public class BitbucketRestClient
```

Comment:
Improvement for the clients would be to handle errors if they occur, so log that a request failed